### DISCUSSION TITLE
Fish Speech running on colab

### DISCUSSION THREAD

**Discussion by mikhail2013ru** [fishaudio/fish-speech#702] [General]
Who can create a notepad to launch fish speech on colab or kaggle? It would be possible to make an inference there, and train models.

**Comment by None1145** [comment]
> Who can create a notepad to launch fish speech on colab or kaggle? It would be possible to make an inference there, and train models.

I am trying to create, please wait a moment.

**Comment by None1145** [comment]
> Who can create a notepad to launch fish speech on colab or kaggle? It would be possible to make an inference there, and train models.

Try this?
https://colab.research.google.com/drive/1trBvrdgyI-Ntd45ZnlT5lhGsI_HnKjC1?usp=sharing

Because version 1.5 might have problems, I used version 1.4.3

**Reply by mikhail2013ru** [reply]
> > Who can create a notepad to launch fish speech on colab or kaggle? It would be possible to make an inference there, and train models.
> 
> Try this? https://colab.research.google.com/drive/1trBvrdgyI-Ntd45ZnlT5lhGsI_HnKjC1?usp=sharing
> 
> Because version 1.5 might have problems, I used version 1.4.3

There are a lot of settings and paths, can you redo it for the 1.5 version? By the way, is it possible to train a model with this notebook, or is it just inference and zero-shot cloning?

**Reply by None1145** [reply]
Both inference and training are available.

To use version 1.5, you just need to replace all occurrences of 1.4 with 1.5 and comment out `!git checkout tags/v1.4.3`.

**Reply by None1145** [reply]
Currently, Fish Speech v1.5 does not support model fine-tuning (SFT) and can only perform inference. If you insist on attempting SFT, you can try using this [repository](https://github.com/None1024Outlook/fish-speech-v1.5-sft-attempt.git). Note that this repository represents an extremely risky training attempt, and you should not spend significant time on it.

**Reply by mikhail2013ru** [reply]
> Currently, Fish Speech v1.5 does not support model fine-tuning (SFT) and can only perform inference. If you insist on attempting SFT, you can try using this [repository](https://github.com/None1024Outlook/fish-speech-v1.5-sft-attempt.git). Note that this repository represents an extremely risky training attempt, and you should not spend significant time on it.

Can you help me train the model in Russian? How many hours is the dataset optimal? Can I contact you via telegram?

**Reply by None1145** [reply]
I can give it a try, but I still recommend waiting for the official 1.5 update from the development team. Generally, the more time for the dataset, the better. You can contact me via Telegram, and I’ll provide whatever help I can.

**Reply by mikhail2013ru** [reply]
> I can give it a try, but I still recommend waiting for the official 1.5 update from the development team. Generally, the more time for the dataset, the better. You can contact me via Telegram, and I’ll provide whatever help I can.

Please write how to contact you in the telegram.

**Reply by None1145** [reply]
Please visit my GitHub homepage to check
